submissions/benjaminKamena/database/query.sql

```diff
@@ -0,0 +1,48 @@
+USE train;
+SHOW TABLES;
+SHOW COLUMNS FROM survival;
+SELECT * FROM survival;
+
+
+-- Q1. What is the total number of passengers who survived?
+SELECT COUNT(*)
+FROM survival
+WHERE survived = 1;
+
+-- After running the query above the answer is 342.
+
+-- Q2. What is the total number of passengers who did not survive?
+SELECT COUNT(*)
+FROM survival
+WHERE survived = 0;
+
+-- After running the query above the answer is 549.
+
+-- Q3. Get the name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg?
+SELECT passenger.name, passenger.sex 
+FROM passenger
+INNER JOIN traveldetail
+ON passenger.PassengerId = traveldetail.PassengerId
+WHERE age < 27 
+AND Embarked = 'Q'
+OR Embarked = 'C';
```

Comment:
This is the right thing to do `WHERE age < 27 
AND (Embarked = 'Q'
OR Embarked = 'C');`